ComponentsInitialization.js

```diff
@@ -1,62 +1,49 @@
 define(function (require) {
-    return function (GEPPETTO) {
-        var React = require('react');
-        var ReactDOM = require('react-dom');
-        var createMuiTheme = require('@material-ui/core/styles/createMuiTheme').default;
-        var MuiThemeProvider = require('@material-ui/core/styles/MuiThemeProvider').default;
-        var HNNMain = require('./HNNMain').default;
-        var Console = require('../../js/components/interface/console/Console');
-        var TabbedDrawer = require('../../js/components/interface/drawer/TabbedDrawer');
-        var PythonConsole = require('../../js/components/interface/pythonConsole/PythonConsole');
-        var execPythonMessage = require('../../js/communication/geppettoJupyter/GeppettoJupyterUtils').execPythonMessage;
+	return function (GEPPETTO) {
+		let React = require('react');
+		let ReactDOM = require('react-dom');
+		let MuiThemeProvider = require('@material-ui/core/styles/MuiThemeProvider').default;
+		let Utils = require('./Utils').default;
+		let HNNMain = require('./HNNMain').default;
+		let theme = require('./Theme').default;
+		let Console = require('../../js/components/interface/console/Console');
+		let TabbedDrawer = require('../../js/components/interface/drawer/TabbedDrawer');
+		let PythonConsole = require('../../js/components/interface/pythonConsole/PythonConsole');
```

Comment:
@rodriguez-facundo is there a specific reason why we're using require here instead of ES6 import? I'd not say there's something wrong in general with require but since we're doing a ES6 refactoring here it would be great to have everything consistent.

Reply:
The right workflow would be: 
1. Refactor Geppetto
2. Refactor extensions

At the moment I wouldn't worry too much about commonjs, amd, es6. This code is pretty much a template for all extensions right now. I agree it would be nice to have everything homogeneously coded but Geppetto is not small and other projects depend on it. We are already in the process of splitting geppetto, bringing some linting and considering some refactoring but it will take time.